src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RecepMessageDateTimeTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import java.time.Instant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class RecepMessageDateTimeTest {
+    private static final Instant WINTER = Instant.parse("2020-03-28T20:58:00Z");
+    private static final Instant SUMMER = Instant.parse("2020-05-28T20:58:00Z");
+
+    @Test
+    void when_toEdifact_and_instantInWinter_expect_edifactIsUTC() throws EdifactValidationException {
+        assertThat(new RecepMessageDateTime(WINTER).toEdifact())
+            .isEqualTo("DTM+815:202003282058:306'");
```

Comment:
I feel it would help if we created a couple of constants here for the edifact string. At first glance, the string looks the same for all tests

Reply:
The receps are different for the summer and winter cases (May or March, respectively). Extracted constants.